components/pages/toast.html

```diff
@@ -0,0 +1,92 @@
+            <li><a href="image.html">Image</a></li>
+            <li><a href="input.html">Input</a></li>
+            <li><a href="navigation.html">Navigation</a></li>
+            <li><a href="rating.html">Rating</a></li>
+            <li><a href="slider.html">Slider</a></li>
+            <li><a href="toast.html">Toast</a></li>
+        </ul>
+    </aside>
+
+    <div class="body-text">
+        <h1>Toast</h1>
+        <p class="description">Toasts are lightweight notifications designed to mimic the push notifications that
+            have been popularized by mobile and desktop operating systems.Generally they are used as an interface
+            feedback pattern for informing the user about the results of an action.
+        </p>
+        <h2>Can check live demo of Toasts below</h2>
+
+        <button class="bt btn-primary" id="top-toast-id">Top Toast</button>
+        <button class="bt btn-primary" id="bottom-toast-id">Bottom Toast</button>
+
+        <div class="toast top-toast" id="top-content">
+            <p class="description toast-element"> This is top toast description
+                <button id="top-toast-close"><i class="fa fa-times"></i></button>
+            </p>
+        </div>
+
+        <div class="toast bottom-toast" id="bottom-content">
+            <p class="description  toast-element">This is bottom toast description
+                <button id="bottom-toast-close"><i class="fa fa-times"></i></button>
+            </p>
+        </div>
+        <p class="description"> Copy the following code to add toast component in your project</p>
+        <iframe class="iframe-box"
+            src="https://carbon.now.sh/embed?bg=rgba%2815%2C118%2C110%2C1%29&t=seti&wt=none&l=auto&width=774&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=false&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cbutton%2520class%253D%2522bt%2520btn-primary%2522%2520id%253D%2522top-toast-id%2522%253ETop%2520Toast%253C%252Fbutton%253E%250A%253Cbutton%2520class%253D%2522bt%2520btn-primary%2522%2520id%253D%2522bottom-toast-id%2522%253EBottom%2520Toast%253C%252Fbutton%253E%250A%250A%253Cdiv%2520class%253D%2522toast%2520top-toast%2522%2520id%253D%2522top-content%2522%253E%250A%2520%253Cp%2520class%253D%2522description%2520toast-element%2522%253E%250A%2520%2520%253Cbutton%2520id%253D%2522top-toast-close%2522%253E%253Ci%2520class%253D%2522fa%2520fa-times%2522%253E%253C%252Fi%253E%253C%252Fbutton%253E%250A%2520%253C%252Fp%253E%250A%253C%252Fdiv%253E%250A%250A%253Cdiv%2520class%253D%2522toast%2520bottom-toast%2522%2520id%253D%2522bottom-content%2522%253E%250A%2520%253Cp%2520class%253D%2522description%2520%2520toast-element%2522%253E%250A%2520%2520%253Cbutton%2520id%253D%2522bottom-toast-close%2522%253E%253Ci%2520class%253D%2522fa%2520fa-times%2522%253E%253C%252Fi%253E%253C%252Fbutton%253E%250A%2520%253C%252Fp%253E%250A%253C%252Fdiv%253E"
+            style="width: 820px; height: 474px;"
```

Comment:
Please avoid inline styles. You can create a class for this and add the class instead.

Reply:
Added stying in class.